modules/datastore.js

```diff
@@ -0,0 +1,29 @@
+const {Datastore} = require('@google-cloud/datastore');
+
+const datastore = new Datastore({
+  projectId: 'cloud-ad-step-2020',
+});
+
+/**
+ * Saves a new entity given a key and a hashmap of parameters.
+ * If key defines only the Kind of the enitity, a new entity will be created.
+ * If key defines both the Kind and the id of an existing entity, it will be updated.
+ *
+ * Creates a new function for datastore to avoid code repetition.
+ **/
+datastore.store = async function(key, parameters) {
+  const entityKey = datastore.isKey(key) ? key : datastore.key(key);
+
+  const entity = {
+    key : entityKey,
+    data: parameters
+  };
+
+  try {
```

Comment:
does this block do anything?

Reply:
Whoops! Looks like it get left over because we were using it to `console.log()` earlier, I'll change it before merging